base/constants.go

```diff
@@ -144,6 +144,8 @@ const (
 	// Intended to be used in Meta Map and related tests
 	MetaMapXattrsKey = "xattrs"
 
+	DocumentXattrKey = "$document"
```

Comment:
```suggestion
	VirtualXattrKey = "$document"
```

But I wonder if we can just pull `RevSeqNoXattrKey` and use `$document.revid` here, and scope this more narrowly.